### DISCUSSION TITLE
Gamemode Selection Menu

### DISCUSSION THREAD

**Discussion by Vuldora** [CorruptedArk/open-chaos-chess#43] [Ideas]
With the increasing amount of options for playing the game (bloodthirst, agressive computer, chess960, etc...) do you think a ganemode selection menu would be fitting for the game? If not do you think picking game options from the options menu is good enough, or do you have a different idea? 

Pleasr share your opnion if you have an idea!

**Comment by CorruptedArk** [comment]
I think it could work just fine, but I don't know if it's better to hide option inside more menus than just to leave them at the top level of settings. I can see an argument for setting headers in the list to separate categories though
